src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      return this.result + number;
+    },
```

Comment:
It will return to `this.result` in 60 line. Just do operation between to parameters. 
![image](https://user-images.githubusercontent.com/70009205/212153721-5d688aa4-e247-4124-bf44-62a77bbcb184.png)
```suggestion
    add(a, b) {
      return a + b;
    },
```

Fix all cases